ip_test.go

```diff
@@ -92,3 +92,87 @@ func TestAccIPReservation(t *testing.T) {
 		t.Fatalf("Reservation %s should be deleted at this point", res)
 	}
 }
+
+func TestAccGlobalIPReservation(t *testing.T) {
+	skipUnlessAcceptanceTestsAllowed(t)
+
+	c, projectID, teardown := setupWithProject(t)
+	defer teardown()
+	quantityToMask := map[int]int{
+		1: 32, 2: 31, 4: 30, 8: 29, 16: 28,
+	}
+
+	quantity := 1
+
+	ipList, _, err := c.ProjectIPs.List(projectID)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if len(ipList) != 0 {
+		t.Fatalf("There should be no reservations a new project, existing list: %s", ipList)
+	}
+
+	req := IPReservationRequest{
+		Type:     "global_ipv4",
+		Quantity: quantity,
+		Comments: "packngo test",
+	}
+
+	res, _, err := c.ProjectIPs.Request(projectID, &req)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if res.CIDR != quantityToMask[quantity] {
+		t.Fatalf(
+			"CIDR prefix length for requested reservation should be %d, was %d",
+			quantityToMask[quantity], res.CIDR)
+	}
+
+	if path.Base(res.Project.Href) != projectID {
+		t.Fatalf("Wrong project linked in reserved block: %s", res.Project.Href)
+	}
+
+	if res.Management {
+		t.Fatal("Management flag of new reservation block must be False")
+	}
+	if res.Facility != nil {
```

Comment:
super minor, but seems like a mix of linebreaks between `if` statements and not, can make consistent?